routes/booking_test.go

```diff
@@ -0,0 +1,422 @@
+	WorkspaceID: "f2188d5e-509f-3086-a031-d86da93a55c4",
+	Cancelled:   false,
+	StartDate:   startBooking4,
+	EndDate:     endBooking4,
+	CreatedBy:   "e99a988a-1d41-3997-8d59-959a48ac24a0",
+}
+var startBooking5, _ = utils.TimeStampToTime("1547683200")
+var endBooking5, _ = utils.TimeStampToTime("1547769599")
+var Booking5 = &model.Booking{
+	ID:          "70747d17-f79f-3c82-88f2-a6ac9c7ba86f",
+	UserID:      "e99a988a-1d41-3997-8d59-959a48ac24a0",
+	WorkspaceID: "62eb266e-740d-3973-9e50-77b0287e3026",
+	Cancelled:   false,
+	StartDate:   startBooking5,
+	EndDate:     endBooking5,
+	CreatedBy:   "e99a988a-1d41-3997-8d59-959a48ac24a0",
+}
+var startBooking6, _ = utils.TimeStampToTime("1548115200")
+var endBooking6, _ = utils.TimeStampToTime("1548374399")
+var Booking6 = &model.Booking{
+	ID:          "d8d1beb0-21e3-3084-b29f-ea2b3c194a2b",
+	UserID:      "e99a988a-1d41-3997-8d59-959a48ac24a0",
+	WorkspaceID: "5e56de3d-2323-372d-897f-23d6037c8581",
+	Cancelled:   false,
+	StartDate:   startBooking6,
+	EndDate:     endBooking6,
+	CreatedBy:   "e99a988a-1d41-3997-8d59-959a48ac24a0",
+}
+var startBooking7, _ = utils.TimeStampToTime("1548547200")
+var endBooking7, _ = utils.TimeStampToTime("1548719999")
+var Booking7 = &model.Booking{
+	ID:          "723ac86b-e0e8-39bd-b407-0c1ced6f2d93",
+	UserID:      "e99a988a-1d41-3997-8d59-959a48ac24a0",
+	WorkspaceID: "aad40cbb-4baf-3931-a5d2-6f98b414182a",
+	Cancelled:   false,
+	StartDate:   startBooking7,
+	EndDate:     endBooking7,
+	CreatedBy:   "e99a988a-1d41-3997-8d59-959a48ac24a0",
+}
+var startBooking11, _ = utils.TimeStampToTime("1548547200")
+var endBooking11, _ = utils.TimeStampToTime("1548719999")
+var Booking11 = &model.Booking{
+	ID:          "2f092deb-3c46-3bf9-9bf9-0a31b7c119b7",
+	UserID:      "e99a988a-1d41-3997-8d59-959a48ac24a0",
+	WorkspaceID: "bb15369d-e6e0-33b8-8b97-1779f8865890",
+	Cancelled:   false,
+	StartDate:   startBooking7,
+	EndDate:     endBooking7,
+	CreatedBy:   "e99a988a-1d41-3997-8d59-959a48ac24a0",
+}
+
+func (suite *AppTestSuite) TestGetOneBookingFail() {
+	t := suite.T()
+	rr := executeReq(t, &testRouteConfig{
+		Method:  http.MethodGet,
+		Body:    nil,
+		Handler: suite.app.GetOneBooking,
+		URL:     fmt.Sprintf("/bookings/%s", "2"),
+		URLParams: map[string]string{
+			"id": "2",
+		},
+	})
+
+	assert.Equal(t, http.StatusInternalServerError, rr.Code, "status code")
+}
+
+func (suite *AppTestSuite) TestGetOneBooking() {
+	bookingId := Booking1.ID
+	t := suite.T()
+	rr := executeReq(t, &testRouteConfig{
+		Method:  http.MethodGet,
+		Body:    nil,
+		Handler: suite.app.GetOneBooking,
+		URL:     fmt.Sprintf("/bookings/%s", bookingId),
+		URLParams: map[string]string{
+			"id": bookingId,
+		},
+	})
+
+	assert.Equal(t, rr.Code, http.StatusOK, "status code")
+	var payload *model.Booking
+	_ = json.Unmarshal(rr.Body.Bytes(), &payload)
+	assert.Equal(t, Booking1, payload, "The patched booking is not the same as the sent request.")
```

Comment:
u can leave the message blank, i dont think the sentence here makes much sense anyways (probably a copy/paste)

Reply:
I just think a non-empty message atleast lets us copy paste it and find where the error was sent